tests/integration.rs

```diff
@@ -5396,6 +5410,125 @@ mod tests {
                     .spawn()
                     .unwrap()
             }
+
+            // XXX Follow up test involving multiple disks will require:
```

Comment:
Why does it need to be ntfs for multiple disks?

Reply:
It's not strictly required for multiple disks. The point is just NTFS will be probably the case in first place, so it should be tested. If we let this test case go by FAT, then NTFS image should be added in the other one.

Thanks